docker-compose.yaml

```diff
@@ -3,32 +3,41 @@ services:
   gateway:
     image: ${BENTOV2_GATEWAY_IMAGE}:${BENTOV2_GATEWAY_VERSION}
     container_name: ${BENTOV2_GATEWAY_CONTAINER_NAME}
+    depends_on:
+      redis:
+        condition: service_healthy
     environment:
       - CHORD_DEBUG=false
       - CHORD_PERMISSIONS=true
       - CHORD_PRIVATE_MODE=${BENTOV2_PRIVATE_MODE}
       - CHORD_URL=${BENTOV2_PORTAL_PUBLIC_URL_TRAILING_SLASH}
+      - CBIOPORTAL_URL=${BENTO_CBIOPORTAL_PUBLIC_URL}/
 
       - OIDC_DISCOVERY_URI=${BENTOV2_AUTH_PUBLIC_URL}${BENTOV2_AUTH_WELLKNOWN_PATH}
       - REDIRECT_AFTER_LOGOUT_URI=${BENTOV2_AUTH_PUBLIC_URL}${BENTOV2_AUTH_REDIRECT_AFTER_LOGOUT_PATH}
       - CLIENT_ID=${BENTOV2_AUTH_CLIENT_ID}
-      - TOKEN_ENDPOINT_AUTH_METHOD=client_secret_post
       - CLIENT_SECRET
+      - TOKEN_ENDPOINT_AUTH_METHOD=client_secret_post
       - BENTOV2_SESSION_SECRET
 
       - BENTOV2_AUTH_DOMAIN
       - BENTOV2_DOMAIN
       - BENTOV2_PORTAL_DOMAIN
+      - BENTOV2_CBIOPORTAL_DOMAIN
 
       - BENTOV2_USE_EXTERNAL_IDP
       - BENTOV2_USE_BENTO_PUBLIC
 
       - BENTOV2_GATEWAY_CONTAINER_NAME
+
       - BENTOV2_GATEWAY_INTERNAL_CERTS_DIR
       - BENTOV2_GATEWAY_INTERNAL_PORTAL_FULLCHAIN_RELATIVE_PATH
       - BENTOV2_GATEWAY_INTERNAL_PORTAL_PRIVKEY_RELATIVE_PATH
       - BENTOV2_GATEWAY_INTERNAL_FULLCHAIN_RELATIVE_PATH
       - BENTOV2_GATEWAY_INTERNAL_PRIVKEY_RELATIVE_PATH
+      - BENTOV2_GATEWAY_INTERNAL_CBIOPORTAL_FULLCHAIN_RELATIVE_PATH
+      - BENTOV2_GATEWAY_INTERNAL_CBIOPORTAL_PRIVKEY_RELATIVE_PATH
```

Comment:
The gateway will not start if these cert files are not found, which is true when `BENTO_CBIOPORTAL_ENABLED=false`